src/MonitoringFunctions/Functions/WorkItemCreator.cs

```diff
@@ -0,0 +1,187 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.TeamFoundation.Core.WebApi;
+using Microsoft.TeamFoundation.WorkItemTracking.WebApi;
+using Microsoft.TeamFoundation.WorkItemTracking.WebApi.Models;
+using Microsoft.VisualStudio.Services.Common;
+using Microsoft.VisualStudio.Services.WebApi;
+using Microsoft.VisualStudio.Services.WebApi.Patch.Json;
+using MonitoringFunctions.Models;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace MonitoringFunctions.Functions
+{
+    internal static class WorkItemCreator
+    {
+        private const string _projectName = "devdiv";
```

Comment:
very minor:  This looks looks like an instance member prefix for a static member.  Consider instead ```ProjectName```

Reply:
https://stackoverflow.com/questions/242534/c-sharp-naming-convention-for-constants